backend/attendance-service/src/main/java/org/egov/models/AttendanceAttendeeSearchCriteria.java

```diff
@@ -1,4 +1,4 @@
-package org.egov.web.models;
+package org.egov.models;
```

Comment:
Remove the `Attendance` prefix for Attendee and Staff models.

Reply:
names refactored.